homeassistant/helpers/aiohttp_client.py

```diff
@@ -133,34 +134,32 @@ def _async_get_connector(hass, verify_ssl=True):
     if verify_ssl:
         if DATA_CONNECTOR not in hass.data:
             connector = aiohttp.TCPConnector(loop=hass.loop)
+            _async_register_connector_shutdown(hass, connector)
```

Comment:
May I suggest you remove both calls to this function and inline this function just before you return out of this function ?